src/ert/scheduler/slurm_driver.py

```diff
@@ -0,0 +1,401 @@
+from __future__ import annotations
+
+import asyncio
+import itertools
+import logging
+import shlex
+import stat
+import tempfile
+import time
+from dataclasses import dataclass
+from enum import Enum, auto
+from pathlib import Path
+from typing import (
+    Iterator,
+    Optional,
+    Tuple,
+)
+
+from ert.scheduler.driver import Driver
+from ert.scheduler.event import Event, FinishedEvent, StartedEvent
+
+logger = logging.getLogger(__name__)
+
+
+class JobStatus(Enum):
+    PENDING = auto()
+    COMPLETED = auto()
+    RUNNING = auto()
+    FAILED = auto()
+    CANCELED = auto()
+    COMPLETING = auto()
+    CONFIGURING = auto()
+
+
+@dataclass
+class JobData:
+    iens: int
+    submitted_timestamp: float
```

Comment:
I don't think we need `submitted_timestamp` in the slurm driver.

Reply:
done